Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -272,6 +299,47 @@ final class RewardCell: UICollectionViewCell, ValueCell {
       |> ksr_addArrangedSubviewsToStackView()
   }
 
+  // MARK: - View controller containment
+
+  func addCollectionViewController(to parent: UIViewController) {
+    _ = self.collectionViewController
+      |> UIViewController.lens.view .. UIView.lens.isHidden .~ true
+
+    parent.addChild(self.collectionViewController)
```

Comment:
For reusability `PillCollectionViewController` is a view controller and in order for it to being properly added to view hierarchy and view-controller hierarchy (UIViewController containment) it has to be added / removed as a `child` to/from a `parent` (this is best practice).

Reply:
This will obviously not be needed when we migrate to `stack view`

Reply:
@dusi do we forsee needing to reuse this UI anywhere? It's seems unnecessary to me to use a view controller since this piece of UI is very static and has no user interaction requirements. I would imagine adding a `UICollectionView` to the `baseStackView` would be sufficient to meet the requirements of this feature.

Reply:
I'd be happy to do that.

My reasoning was that for something like this `pill layout` which is easily done with `flex box` but not really with auto layout it seemed like a good opportunity to introduce this as a reusable component.

Any thoughts @justinswart ? If we use a `collection view` directly we can get rid of the VC containment fiddling as well.